client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -30,6 +31,23 @@ class StoreStats extends Component {
 			date: today,
 			quantity: '30'
 		};
+		const topSellersQuery = {
+			unit,
+			date: selectedDate,
+			quantity: '7',
+			limit: '3',
+		};
+		const translate = this.props.translate;
+		const moduleStrings = {
+			title: translate( 'Products', { context: 'Stats: title of module' } ),
+			item: translate( 'Title', { context: 'Stats: module row header for product title.' } ),
+			value: translate( 'Qty', { context: 'Stats: module row header for number of products sold.' } ),
+			values: [
+				{ value: translate( 'Qty', { context: 'Stats: module row header for number of products sold.' } ) },
+				{ value: translate( 'Total', { context: 'Stats: module row header for total amount sold.' } ) },
```

Comment:
Hi! I've found a possible matching string that has already been translated **38** times:
`translate( 'Total:', { context: 'Upgrades: Total cart cost in checkout when buying a full price upgrade'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woo-stats-topSellers)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?